heroku/templates/src/main/java/package/config/_HerokuDatabaseConfiguration.java

```diff
@@ -32,32 +32,22 @@ public void setEnvironment(Environment environment) {
     public DataSource dataSource(<% if (hibernateCache == 'hazelcast') { %>CacheManager cacheManager<% } %>) {
         log.debug("Configuring Heroku Datasource");
 
-        String herokuUrl = propertyResolver.getProperty("heroku-url");
+        String herokuUrl = System.getenv("JDBC_DATABASE_URL");
```

Comment:
I don't have a strong opinion on this either! You can leave it like this if you feel it's better, you're the one maintaining this in the end :-)